components/protocol/component.go

```diff
@@ -7,15 +7,15 @@ import (
 	"github.com/labstack/gommon/bytes"
 	"go.uber.org/dig"
 
+	p2ppeer "github.com/libp2p/go-libp2p/core/peer"
```

Comment:
why the custom import name? there should be no collision with hive.go anymore if you use the default `peer`